DVSRegister/Views/Register/ProviderDetails.cshtml

```diff
@@ -132,7 +132,35 @@
                             </div>
                         }
 
-                       
+
+                        @if (service.ServiceSupSchemeMapping != null)
```

Comment:
@if (service.ServiceSupSchemeMapping != null)    this check would be enough to check the ServiceSupSchemeMapping  exists. HasSupplementary field will be always true if ServiceSupSchemeMapping   is not null, can be removed